packages/domain-events/src/Domain/Event/SubscriptionRenewedEvent.ts

```diff
@@ -0,0 +1,8 @@
+import { DomainEventInterface } from './DomainEventInterface'
+
+import { SubscriptionCancelledEventPayload } from './SubscriptionCancelledEventPayload'
+
+export interface SubscriptionCancelledEvent extends DomainEventInterface {
```

Comment:
@karolsojko just noticed SubscriptionCancelledEvent and SubscriptionRenewedEvent are each on the other's corresponding file

Reply:
ah my bad - copy pasting :) will fix